app.config.js

```diff
@@ -2,7 +2,7 @@
 import { convention } from './convention.config.json'
 
 const urlMatcher = /^([^:]+):\/\/([^/]+)(\/.*)$/
-const [, appBaseProtocol, appBaseHost, appBasePath] = [...convention.appBase.match(urlMatcher)]
+const [appBaseProtocol, appBaseHost, appBasePath] = [...convention.appBase.match(urlMatcher)]
```

Comment:
The leading comma is significant, as the 0-th group is the entire string if matched and the 1,2,3rd groups are the matched regex segments.